pkg/oc/cli/cmd/deploy.go

```diff
@@ -108,6 +108,8 @@ func NewCmdDeploy(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		Long:       fmt.Sprintf(deployLong, fullName),
 		Example:    fmt.Sprintf(deployExample, fullName),
 		SuggestFor: []string{"deployment"},
+		Deprecated: "Use the oc rollout latest and oc rollout cancel",
```

Comment:
oc rollout status instead of cancel (oc deploy will wait for the deployment to finish, the equivalent for waiting is oc rollout status)